01week/datatypes.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+//using console logs to test
+
+//simple date and time
+const day = new Date();
```

Comment:
This method is fine, however the instructions were to 'write a program' which means 'write a function'. Please wrap in a function.